fabs/src/main/java/com/decathlon/vitamin/compose/fabs/VitaminFabsExtended.kt

```diff
@@ -0,0 +1,135 @@
+package com.decathlon.vitamin.compose.fabs
+
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.sizeIn
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.FloatingActionButton
+import androidx.compose.material.FloatingActionButtonDefaults
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Add
+import androidx.compose.material.ripple.LocalRippleTheme
+import androidx.compose.material.ripple.RippleTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.graphics.vector.rememberVectorPainter
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.decathlon.vitamin.compose.foundation.VitaminTheme
+
+object VitaminFabsExtended {
+
+    /**
+     * The FAB extended must be used for primary, positive actions
+     * @param text The text inside the FAB extended
+     * @param modifier The [Modifier] to be applied to the component
+     * @param enabled True if you can click on the FAB extended, otherwise false
+     * @param icon The optional icon to be displayed inside the FAB extended on the right left side of the text
+     * @param colors The colors of the background and the content in enabled and disabled
+     * @param sizes The sizes for the icon, the text and the component itself
+     * @param ripple The ripple effect applied on the component
+     * @param onClick The callback to be called when the user click on the FAB extended
+     */
+    @Composable
+    fun Primary(
+        text: String,
+        modifier: Modifier = Modifier,
+        enabled: Boolean = true,
+        icon: Painter? = null,
+        colors: FabsColors = VitaminFabsColors.primary(),
+        sizes: FabExtendedSizes = VitaminFabExtendedSizes.default(),
+        ripple: RippleTheme = VitaminTheme.ripples.brand,
+        onClick: () -> Unit
+    ) {
+        CompositionLocalProvider(LocalRippleTheme provides ripple) {
```

Comment:
After design review, we prefer not overriding ripple effect because Material one is more accessible. 
You can remove CompositionLocalProvider 

Don't forget to update documentation & readme

Reply:
done